src/shared/capture/capturev4l.cpp

```diff
@@ -884,18 +877,15 @@ void CaptureV4L::writeParameterValues(VarList * item) {
         return;
     }
     VarInt * vint=0;
-    VarBool * vuseabs=0;
     VarBool * venabled=0;
-    VarBool * vauto=0;
     VarBool * vwasread=0;
     VarBool * vdefault=0;
     
     vector<VarType *> children=item->getChildren();
     for (unsigned int i=0;i<children.size();i++) {
         if (children[i]->getType()==VARTYPE_ID_BOOL && children[i]->getName()=="was_read") vwasread=(VarBool *)children[i];
         if (children[i]->getType()==VARTYPE_ID_BOOL && children[i]->getName()=="enabled") venabled=(VarBool *)children[i];
-        if (children[i]->getType()==VARTYPE_ID_BOOL && children[i]->getName()=="auto") vauto=(VarBool *)children[i];
-        if (children[i]->getType()==VARTYPE_ID_BOOL && children[i]->getName()=="default") vauto=(VarBool *)children[i];
+        if (children[i]->getType()==VARTYPE_ID_BOOL && children[i]->getName()=="default") vdefault=(VarBool *)children[i];
```

Comment:
vdefault is somehow used to reset vint. Not sure what's exactly happening there.